internal/app/components/terms/termRecords.go

```diff
@@ -35,3 +44,8 @@ func isPrereqSatisfied(record *courses.CourseRecord, pastRecords *courses.Course
 	}
 	return record.Prereqs.IsSatisfied(pastRecords)
 }
+
+// TODO: implement
+func GetTermRecordById(id TermRecordId) TermRecord {
+	return nil
```

Comment:
I don't think nil is a `TermRecord` type, maybe make the return value `*TermRecord` since pointers are nullable